### PR TITLE
[4.1]Recent Articles admin module - workflows

### PR DESCRIPTION
**Pull request by brianteeman**
## Purpose
If you are using workflows then it is useful to know what stage an article is at. This PR adds the display of the stage to the administrator module "recently added articles". The column is only displayed when workflows is enabled

### Test Setup Instructions

- Enable workflows in the content options
- Create several articles and set them at different stages
- check the module on the home dashboard and the content dashboard

### Before this PR
No visible means to see the stage of all the articles
image

### After this PR
The stage is now displayed if workflows is enabled
image

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :red_circle: unsuccessfully on 429ba19ad4c194cc07c28508885065aeec0c4e23<br /><br />Tested in Joomla 4.1.3-rc2-dev of 6 May in Wampserver 3.2.8 using PHP 8.0.15.

When the default stage is used, the language string COM_WORKFLOW_BASIC_STAGE is not parsed in Recently Added Articles -screenshot 1.  Editing the entry of Basic Stage  did not change the text -screenshot 2.  The names of the stages in the list of articles in the Content Dashboard have been parsed all right - screenshot 3.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37749">issues.joomla.org/tracker/joomla-cms/37749</a>.</sub>

**Comment by toivo** [comment]
screen shot 2022-05-06 at 05 40 06screen shot 2022-05-06 at 05 40 06screen shot 2022-05-06 at 05 40 06<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37749">issues.joomla.org/tracker/joomla-cms/37749</a>.</sub>

**Comment by brianteeman** [comment]
@toivo Good spot. Should be fixed now

File: administrator/modules/mod_latest/mod_latest.php [hunk 1]
```diff
@@ -9,12 +9,19 @@
...
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Component\ComponentHelper;
 use Joomla\CMS\Helper\ModuleHelper;
 use Joomla\CMS\Layout\LayoutHelper;
 use Joomla\Module\Latest\Administrator\Helper\LatestHelper;
 
 $model = $app->bootComponent('com_content')->getMVCFactory()->createModel('Articles', 'Administrator', ['ignore_request' => true]);
 $list = LatestHelper::getList($params, $model);
+$workflow_enabled  = ComponentHelper::getParams('com_content')->get('workflow_enabled');
```
Comment (sandewt) [inline administrator/modules/mod_latest/mod_latest.php]: One space to much, after enabled  :-)

Should be:
`$workflow_enabled = ComponentHelper::`

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on c8404fe4f0fa67de947a4d70e84299012e15a84b<br /><br />Tested successfully in Joomla 4.1.3-rc2-dev of 6 May.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37749">issues.joomla.org/tracker/joomla-cms/37749</a>.</sub>

File: administrator/modules/mod_latest/tmpl/default.php [hunk 2]
```diff
@@ -40,6 +43,11 @@
 					<?php echo htmlspecialchars($item->title, ENT_QUOTES, 'UTF-8'); ?>
 				<?php endif; ?>
 			</th>
+			<?php if ($workflow_enabled) : ?>
+				<td>
+					<?php echo Text::_($item->stage_title); ?>
+				</td>
+				<?php endif; ?>
```
Comment (Quy) [inline administrator/modules/mod_latest/tmpl/default.php]:
```suggestion
			<?php endif; ?>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5077c71b101923fb04d37567ced6b65afa2b7ba3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37749">issues.joomla.org/tracker/joomla-cms/37749</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37749">issues.joomla.org/tracker/joomla-cms/37749</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**